### PR TITLE
Feature/214 use participation list

### LINKED ISSUE #214: Use participant list
API doesn't yet support participant list even though it's shown in reservation form. When API supports it, send participant list to API and also show participant list on reservation info modal.

Also show participants in reservation info modal. Use component FormattedUserText.

### PR DESCRIPTION
**Pull request by YoYuUm**
Closes #214

I've tried to create a reservation sending participants. But after fetching the reservation, this one does not contain the participants field

### CODE REVIEW

**Comment by evehmane** [comment]
The code looks good. I have to ask backend developers about the support for `participants` field. They said it's done but apparently it's not.

**Comment by evehmane** [comment]
- [ ] I just found an old mockup where the participant list is at the bottom of the modal. Could you please change this. Sorry I didn't realize this before!
nayttokuva 2017-06-11 kello 19 50 20

**Comment by evehmane** [comment]
Because of some mistake, backend doesn't have that field yet. But they will fix it at some point. I think I'll merge this after you have done that one change. Later we can test this feature with browser when backend is done.

File: app/shared/modals/reservation-info/ReservationInfo.js [hunk 1]
```diff
@@ -108,6 +104,13 @@ export default function ReservationInfoModal(props) {
           name="Varauksen viestit"
           reservationId={reservation.id}
         />
+        <hr />
+        <Row>
+          <Col className="details-row reservation-participants" xs={6}>
```
Comment (evehmane) [inline app/shared/modals/reservation-info/ReservationInfo.js]: - [ ] This should take the whole modal width as in the mockup, instead of half of it.

**Comment by YoYuUm** [comment]
I can do the FormattedUserText/WrappedText in another issue as it seems it wouldn't be the scope of this PR

File: app/shared/modals/reservation-info/ReservationInfo.js (hunk 1 above)
Comment (YoYuUm) [inline app/shared/modals/reservation-info/ReservationInfo.js]: Fixed in commit.

**Comment by evehmane** [comment]
Ok, that's fine!

**Pull request merged**